<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menuWidget</class>
 <widget class="QWidget" name="menuWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_make">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Make a Account</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_deposit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Deposit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_withdraw">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Withdraw</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_see">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>See all Accounts</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_specific">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Specific Inquiry</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
